code/modules/admin/topic.dm

```diff
@@ -911,6 +911,12 @@
 		else
 			dat += "<td width='20%'><a href='?src=[REF(src)];[HrefToken()];jobban3=[ROLE_MIND_TRANSFER];jobban4=[REF(M)]'>Mind Transfer Potion</a></td>"
 
+		//Hulk
+		if(jobban_isbanned(M, ROLE_HULK))

```

Comment:
This is copypasted. Undo the copypaste here and clean up this proc.

Reply:
Refer to above comment